Sources/SmithyReadWrite/Document.swift

```diff
@@ -1,7 +1,16 @@
-/*
- * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0.
- */
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import struct Foundation.Data
+import class Foundation.JSONSerialization
+import class Foundation.NSNull
+import class Foundation.NSNumber
+import func CoreFoundation.CFGetTypeID
+import func CoreFoundation.CFBooleanGetTypeID
 
 public enum Document {
```

Comment:
The `Document` type is moved from ClientRuntime to SmithyReadWrite, and now converts itself to/from a JSON object instead of being Codable.